Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.10 AS builder
+LABEL builder=true
+
+ENV CGO_ENABLED=0
```

Comment:
```suggestion
ENV CGO_ENABLED=0 GOPATH=/go
```

Reply:
Do we need this?

Reply:
It looks nicer and is easier to convert it to an ``export ...``.